### ISSUE TITLE
Hammer Editor for ZMR (item_ammo_smg1 crash problem)

### ISSUE THREAD

**Issue by xDadiKx** [zm-reborn/zmr-game#17]
Trying to add ammo for MAC10. Using item_ammo_smg1.
With the entity tool, I am trying to add item_ammo_smg1 (mac10 ammo) to my map but after left clicking, it literally crashes Hammer Editor all together.
I can add Mac10 ammo using item crates, so the player breaks them and the player gets the ammo without crashing. That is a solution but I am trying to add it independently, on the floor without any use of crates.

**Edit**
Just noticed this also crashes when using item_ammo_ar2.

**Comment by Fichtendummkopf** [comment]
Workaround:
Add the hl2dm.fgd to your ZMR config.

My ZMR Hammer config crashes on loading some of my old ZM map vmf files, and also when placing Mac10 ammo.

Adding the ZMR fgd to the HL2DM hammer config does not have any of those 2 problems.
(I just changed the Game directory in the HL2DM config to $SteamDir\SteamApps\SourceMods\zombie_master_reborn ,added the ZMR .fgd in addition to the hl2mp.fgd and changed the directory to palce the compiled maps.)

I just now tried adding the hl2dm.fgd to the ZMR hammer config alongside the ZMR fgd, and now I can place item_ammo_smg1 without hammer crashing.

Maybe the ZMR .fgd misses some stuff the hl2mp.fgd or the @included hl2.fgd does?
